src/lib.rs

```diff
@@ -1,3 +1,16 @@
+#![cfg_attr(
+  feature = "cargo-clippy",
+  allow(
+      clippy::derive_partial_eq_without_eq,
+      clippy::use_self,
+      clippy::equatable_if_let,
+      clippy::option_if_let_else,
+      clippy::match_like_matches_macro,
+      clippy::comparison_chain,
+      clippy::if_same_then_else,
+      clippy::option_if_let_else,
+  )
+)]
```

Comment:
Would you suggest removing `derive_partial_eq_without_eq`, `equatable_if_let` and/or `match_like_matches_macro`?

(also `option_if_let_else` is there twice)

Reply:
`use_self`, `equatable_if_let` and `option_if_let_else` should be allowed by default since they have the `nursery` level

Reply:
@probablykasper I sorted the lints and removed the `nursery` lints.

As for the the `derive_partial_eq_without_eq`, I'm agnostic on it, but you may be know circumstances that apply to cpc.

`equatable_if_let` is moot since its a nursery level lint.

`match_like_matches_macro` made sense for the `is_numeric_str` fn, but I thought it was best to leave `is_word_char_str` alone for performance reasons.

Reply:
Alright, I'd say the that just `derive_partial_eq_without_eq` should be removed.

Other than that, could you delete the lines instead of just commenting them? Then we're good, ty!

Reply:
Here you go! Looking forward to using cpc to enable qsv's upcoming `math` command 😄

Reply:
Oh, cool!